src/main/java/frc/robot/Robot.java

```diff
@@ -105,6 +108,28 @@ public void teleopInit() {
    */
   @Override
   public void teleopPeriodic() {
+    String gameData;
+    gameData = DriverStation.getInstance().getGameSpecificMessage();
+    if (gameData.length() > 0) {
```

Comment:
Needs a null check and can be written more clearly as:
`if (!(gameData == null || gameData.isEmpty()) {`